src/Routes/Books/Books.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import ActionsBook from './ActionsBook';
+import AddBook from './AddBook';
+
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      genre: 'Action',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      genre: 'Science Fiction',
+      author: 'Frank Herbert',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      genre: 'Economy',
+      author: 'Suzanne Collins',
+    },
+  ];
+  return (
+    <>
+      <div className="container border rounded my-3 p-5">
+        {books.map((book) => (
+          <div className="card mb-3" key={book.id}>
+            <div className="card-body">
+              <h6 className="card-title">{book.genre}</h6>
+              <h4 className="card-text">{book.title}</h4>
+              <p className="text-muted">{book.author}</p>
+              <ActionsBook />
+            </div>
+          </div>
```

Comment:
- [ ] Nice job utilizing reusable components. :+1: Let's work on some minor details to achieve the project requirements. Kindly extract these lines into their own component and pass the data as `props` as suggested in the requirements. ✍️

**Project Requirements**
![Screenshot from 2023-02-24 10-33-43](https://user-images.githubusercontent.com/17243325/221120383-b8d6c97f-d9fa-45af-a0d3-73b1019f229f.png)

Reply:
Thanks for adding this information